src/SFA.DAS.ApprenticeFeedback.Application.UnitTests/Commands/WhenHandlingCreateApprenticeFeedbackCommand.cs

```diff
@@ -0,0 +1,70 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.ApprenticeFeedback.Application.Commands.CreateApprenticeFeedback;
+using SFA.DAS.ApprenticeFeedback.Domain.Entities;
+using SFA.DAS.ApprenticeFeedback.Domain.Interfaces;
+using SFA.DAS.ApprenticeFeedback.Domain.Models;
+using SFA.DAS.Testing.AutoFixture;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeFeedback.Application.UnitTests.Commands
+{
+    public class WhenHandlingCreateApprenticeFeedbackCommand
+    {
+        [Test, MoqAutoData]
+        public async Task And_GetFeedbackTargetIsNull_Then_ThrowException
+            (CreateApprenticeFeedbackCommand command,
+            [Frozen] Mock<IApprenticeFeedbackRepository> mockApprenticeFeedbackRepository,
+            CreateApprenticeFeedbackHandler handler)
+        {
+            mockApprenticeFeedbackRepository.Setup(s => s.GetApprenticeFeedbackTargetById(command.ApprenticeId)).Throws(new Exception());
```

Comment:
I'm not sure this is right, should the mock setup not return null, to allow the exception to be thrown from within the handler?

When asserting on the exception, you can also do

```result.should().throwasync<exception>().WithMessage``` to help tighten the assertion to make sure it's the right exception. If you were to add that now, I don't think it would pass.